arcade/arcade/cli/main.py

```diff
@@ -702,6 +703,59 @@ def dashboard(
         )
 
 
+@cli.command(help="Generate Toolkit documentation", rich_help_panel="Tool Development")
+def generate_toolkit_docs(
+    toolkit_name: str = typer.Option(
+        None, "--toolkit-name", "-n", help="The name of the toolkit to generate documentation for."
+    ),
+    docs_section: str = typer.Option(
+        "",
+        "--docs-section",
+        "-s",
+        help=(
+            "The section of the docs to generate documentation for. E.g. 'productivity', 'sales'. "
+            "Defaults to an empty string (generate the docs in the root of /pages/toolkits)"
+        ),
+        show_default=True,
+    ),
+    docs_root_dir: str = typer.Option(
+        "~/arcade/Team/docs",
+        "--docs-root-dir",
+        "-r",
+        help="The path to the documentation root directory.",
+        show_default=True,
+    ),
+    engine_base_url: str = typer.Option(
+        "http://127.0.0.1:9099",
```

Comment:
[nit] I understand you are re-using the swagger endpoint to get the info you need, but it seems a little odd to have to run the service to get the annotation info.  Would there be a way to pass a directory path to the tool instead, and then look for methods with the `@tool` annotation?  I think could be possible with `inspect` and `ast`

Reply:
the worker certainly has some method to collect this data, but I was too lazy to look for it 🙈

it was faster to get it from the engine.. I'll look for a simple way to get it without the need for running the engine